### PR TITLE
Acrntrace refactor s3

### PR DESCRIPTION
**Pull request by KaigeFu**

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

**Comment by lijinxia** [comment]
Verified+1

File: tools/acrntrace/README.rst [hunk 1]
```diff
@@ -18,8 +18,11 @@ output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
 Options:
 
 -h			print this message
--t period		specify polling interval in milliseconds [1-999]
+-i period		specify polling interval in milliseconds [1-999]
+-t			max time to capture trace data (in second)
```
Comment (dbkinder) [inline tools/acrntrace/README.rst]: If the -t option has a parameter you should mention that:

    -t time            max time to capture trace data (in seconds)

File: tools/acrntrace/README.rst [hunk 2]
```diff
@@ -18,8 +18,11 @@ output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
...
 -c			clear the buffered old data
+-r			minimal amount (in MB) of free space kept on the disk
```
Comment (dbkinder) [inline tools/acrntrace/README.rst]: same here, mention the parameter:

    -r space          amount of free space (in MB) remaining on the disk
                      before acrntrace stops

**Comment by dbkinder** [comment]
@lijinxia This PR was merged before a review of documentation was made (and there were problems with the documentation changes)  :(

**Comment by KaigeFu** [comment]
@dbkinder Should I make another patch to fix the doc problems? If so, I am willing to do it.

**Comment by dbkinder** [comment]
I just did #607

**Comment by KaigeFu** [comment]
@dbkinder Thanks.

**Pull request merged**